core/src/main/java/com/linecorp/armeria/server/http/auth/HttpAuthServiceBuilder.java

```diff
@@ -30,73 +32,53 @@
 import com.linecorp.armeria.server.Service;
 
 /**
- * Builds a new {@link HttpAuthService}.
+ * Builds a new {@link HttpAuthService}. {@link Authorizer}s are executed without short-circuiting -
+ * all authorizers will be executed regardless of the result of any individual one.
```

Comment:
Cancel seems complicated and would generally have hard-to-predict behavior I think.

I actually did have my own reservations about executing all the authorizers, so I've pushed a change that instead runs them in order (but still asynchronously). This makes the behavior of multiple authorizers the same as before.

I guess it's fine since I would expect it to be very rare for a service to have multiple asynchronous authorizers that want to be run in parallel, but let me know what you think.